gpu-lab.md

```diff
@@ -7,21 +7,41 @@ title: Welcome to the CHTC GPU Lab
 The CHTC GPU Lab is a [UW2020-funded project][uw2020] that will expand shared GPU 
 computing infrastructure at UW-Madison.  It will include:
 
-* a pool of shared GPU servers managed by CHTC 
-* a community of technical experts
-* a library of sharable software and documentation
+<table>
+  <tr>
+    <th width="25%">Hardware</th>
+    <th width="25%">Expertise</th>
+    <th width="25%">Documentation</th>
+  </tr>
+  <tr align="center">
+    <td><img width="60%" src="guide-icons/noun_gpu_2528527.png"></td>
+    <td><img width="60%" src="guide-icons/noun_people_1188645.png"></td>
+    <td><img width="60%" src="guide-icons/noun_open book_1179297.png"></td>
+  </tr>
+  <tr>
+    <td>a pool of shared GPU servers managed by CHTC </td>
+    <td>a community of technical experts</td>
+    <td>a library of sharable software and documentation</td>
+  </tr>
+</table>
 
-To get started with GPU computing in CHTC, visit the [GPU jobs guide][gpu-jobs] 
-and see the [runnable examples on GitHub][gpu-examples].
+## Get Involved
 
-For more information about the CHTC GPU Lab contact [Anthony Gitter][gitter]. Email 
-[chtc@cs.wisc.edu](mailto:chtc@cs.wisc.edu) for general CHTC GPU computing 
-questions.  Subscribe to the CHTC GPU Lab WiscList 
+If you want to use GPU resources in CHTC for your research: 
+
+- Apply for a CHTC account if you don't already have one: [Account Request Page][account]
+- See what GPUs are available and how to use them in the [GPU jobs guide][gpu-jobs].
+- For more extended examples, see the following [runnable examples on GitHub][gpu-examples].
+
+Subscribe to the CHTC GPU Lab WiscList 
```

Comment:
Do you think we should clarify who is invited to subscribe to this list?  It is intended for CHTC users, or anyone who is eligible to become a CHTC user.  It currently requires admin approval to join.

Reply:
Yes! It wasn't clear to me from the original document who should apply and what the purpose of the list was. If you can expand or create more of a blurb, that would be great.

Reply:
Let's expand to
```
The CHTC GPU Lab WiscList is used to announce new GPU hardware availability and GPU-related events, solicit feedback from GPU users, and share best practices for GPU computing in CHTC.
Any CHTC user can subscribe to the list by emailing [join-chtc-gpu-lab@lists.wisc.edu](mailto:join-chtc-gpu-lab@lists.wisc.edu).
Their subscription request will be reviewed by the list administrators.
```

Is that more clear?  The last sentence may not be needed and can be cut if you prefer.  Currently the list is only used for one way announcements from the admins.  It the future, we might open it up for user-to-user discussions.

Reply:
@ChristinaLK could you please make this edit?  It's hard for me to use the GitHub suggestion tool to correct make a multi-line change.  After that, could you please merge some or all of these changes?  There will be a lot of traffic to this page during the related Jan 14 event.  Thanks!

Reply:
Yes! I'll get this done this morning.